plan/plans/perf.go

```diff
@@ -0,0 +1,1015 @@
+	rfs = append(rfs, buildEnumResultField(tbName, "ENABLED", []string{"YES", "NO"}, mysql.NotNullFlag, nil))
+	return rfs
+}
+
+// CREATE TABLE if not exists performance_schema.setup_timers (
+// 		NAME			VARCHAR(64) NOT NULL,
+// 		TIMER_NAME		ENUM('NANOSECOND','MICROSECOND','MILLISECOND') NOT NULL);
+func buildResultFieldsForSetupTimers() (rfs []*field.ResultField) {
+	tbName := tableSetupTimers
+	rfs = append(rfs, buildUsualResultField(tbName, "NAME", mysql.TypeVarchar, 64, mysql.NotNullFlag, nil))
+	rfs = append(rfs, buildEnumResultField(tbName, "TIMER_NAME", []string{"NANOSECOND", "MICROSECOND", "MILLISECOND"}, mysql.NotNullFlag, nil))
+	return rfs
+}
+
+// CREATE TABLE if not exists performance_schema.events_statements_current (
+// 		THREAD_ID		BIGINT(20) UNSIGNED NOT NULL,
+// 		EVENT_ID		BIGINT(20) UNSIGNED NOT NULL,
+// 		END_EVENT_ID	BIGINT(20) UNSIGNED,
+// 		EVENT_NAME		VARCHAR(128) NOT NULL,
+// 		SOURCE			VARCHAR(64),
+// 		TIMER_START		BIGINT(20) UNSIGNED,
+// 		TIMER_END		BIGINT(20) UNSIGNED,
+// 		TIMER_WAIT		BIGINT(20) UNSIGNED,
+// 		LOCK_TIME		BIGINT(20) UNSIGNED NOT NULL,
+// 		SQL_TEXT		LONGTEXT,
+// 		DIGEST			VARCHAR(32),
+// 		DIGEST_TEXT		LONGTEXT,
+// 		CURRENT_SCHEMA	VARCHAR(64),
+// 		OBJECT_TYPE		VARCHAR(64),
+// 		OBJECT_SCHEMA	VARCHAR(64),
+// 		OBJECT_NAME		VARCHAR(64),
+// 		OBJECT_INSTANCE_BEGIN	BIGINT(20) UNSIGNED,
+// 		MYSQL_ERRNO		INT(11),
+// 		RETURNED_SQLSTATE		VARCHAR(5),
+// 		MESSAGE_TEXT	VARCHAR(128),
+// 		ERRORS			BIGINT(20) UNSIGNED NOT NULL,
+// 		WARNINGS		BIGINT(20) UNSIGNED NOT NULL,
+// 		ROWS_AFFECTED	BIGINT(20) UNSIGNED NOT NULL,
+// 		ROWS_SENT		BIGINT(20) UNSIGNED NOT NULL,
+// 		ROWS_EXAMINED	BIGINT(20) UNSIGNED NOT NULL,
+// 		CREATED_TMP_DISK_TABLES	BIGINT(20) UNSIGNED NOT NULL,
+// 		CREATED_TMP_TABLES		BIGINT(20) UNSIGNED NOT NULL,
+// 		SELECT_FULL_JOIN		BIGINT(20) UNSIGNED NOT NULL,
+// 		SELECT_FULL_RANGE_JOIN	BIGINT(20) UNSIGNED NOT NULL,
+// 		SELECT_RANGE	BIGINT(20) UNSIGNED NOT NULL,
+// 		SELECT_RANGE_CHECK		BIGINT(20) UNSIGNED NOT NULL,
+// 		SELECT_SCAN		BIGINT(20) UNSIGNED NOT NULL,
+// 		SORT_MERGE_PASSES		BIGINT(20) UNSIGNED NOT NULL,
+// 		SORT_RANGE		BIGINT(20) UNSIGNED NOT NULL,
+// 		SORT_ROWS		BIGINT(20) UNSIGNED NOT NULL,
+// 		SORT_SCAN		BIGINT(20) UNSIGNED NOT NULL,
+// 		NO_INDEX_USED	BIGINT(20) UNSIGNED NOT NULL,
+// 		NO_GOOD_INDEX_USED		BIGINT(20) UNSIGNED NOT NULL,
+// 		NESTING_EVENT_ID		BIGINT(20) UNSIGNED,
+// 		NESTING_EVENT_TYPE		ENUM('TRANSACTION','STATEMENT','STAGE'),
+// 		NESTING_EVENT_LEVEL		INT(11));
+func buildResultFieldsForStmtsCurrent() (rfs []*field.ResultField) {
+	tbName := tableStmtsCurrent
+	rfs = append(rfs, buildUsualResultField(tbName, "THREAD_ID", mysql.TypeLonglong, 20, mysql.NotNullFlag, nil))
+	rfs = append(rfs, buildUsualResultField(tbName, "EVENT_ID", mysql.TypeLonglong, 20, mysql.NotNullFlag, nil))
+	rfs = append(rfs, buildUsualResultField(tbName, "END_EVENT_ID", mysql.TypeLonglong, 20, 0, nil))
```

Comment:
I see from your comment, `END_EVENT_ID	BIGINT(20) UNSIGNED`, but seems have not processed UNSIGNED flag?